lib/global_registry_bindings/entity/push_relationship_methods.rb

```diff
@@ -14,10 +14,17 @@ def relationship
         end
 
         def push_relationship_to_global_registry
-          # Delete relationship if it exists and the related id_value is missing
-          if relationship.related.nil? && relationship.related_id_value.nil? && relationship.id_value
-            delete_relationship_from_global_registry(false)
-            return
+          # We can't push relationship if related model is missing, but we may need to delete
+          if relationship.related.nil?
+            if relationship.related_id_value.nil? && relationship.id_value
+              # Delete relationship if it exists and the related id_value is missing
+              delete_relationship_from_global_registry(false)
+              return
+            elsif relationship.related_binding == :entity
```

Comment:
You don't need the `els` here. `return if ..`

Reply:
Fixed, and moved to it's own method.